### PR TITLE
search issues assigned to me

### PR DESCRIPTION
**Pull request by macbookandrew**
## Description

Adds a new “assigned to me” filter to the “Search Issues” command

## Screencast

SCR-20241015-igsk

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by macbookandrew** [comment]
What do you all think of making the “assigned to me” the top aka default choice, rather than “created by me”?

**Comment by flybynight27** [comment]
> What do you all think of making the “assigned to me” the top aka default choice, rather than “created by me”?

I love that idea. Making it configurable would be better, but for me "assigned to me" is _way_ more useful than "created by me".

File: extensions/gitlab/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # GitLab Changelog
 
+## [Add Issues Assignment Filter] - 2024-10-15
```
Comment (jfkisafk) [inline extensions/gitlab/CHANGELOG.md:3]:
```suggestion
## [Add Issues Assignment Filter] - {PR_MERGE_DATE}
```

File: extensions/gitlab/src/components/issues_search.tsx [hunk 2]
```diff
@@ -53,6 +53,7 @@ export function SearchMyIssues(): JSX.Element {
       searchBarAccessory={
         <List.Dropdown tooltip="Scope" onChange={setScope}>
```
Comment (jfkisafk) [inline extensions/gitlab/src/components/issues_search.tsx:54]:
```suggestion
        <List.Dropdown tooltip="Scope" onChange={setScope} storeValue>
```

**Review by jfkisafk — APPROVED** [review]
LGTM! I think we can storeValue for the accessory so that user don't need to reselect with the next render (replaces using a preference to some degree).

**Review by jfkisafk — APPROVED** [review]

**Review by thomaslombart — APPROVED** [review]

**Pull request merged**